integration/docker_test.go

```diff
@@ -1,15 +1,140 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"os/exec"
 	"time"
 
+	"github.com/docker/docker/opts"
+	"github.com/docker/docker/pkg/namesgenerator"
+	"github.com/fsouza/go-dockerclient"
 	checker "github.com/vdemeester/shakers"
 	check "gopkg.in/check.v1"
 )
 
+var (
+	// Label added to started container to identify them as part of the integration test
+	TestLabel = "io.traefik.test"
+
+	// Images to have or pull before the build in order to make it work
+	// FIXME handle this offline but loading them before build
```

Comment:
As the `FIXME` states, should be handled later in the same fashion as in `docker/docker`, i.e. downloading images on initial build and loading them at runtime :wink:.